getting-started/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id 'java'
+    id 'io.quarkus.gradle.plugin' version '0.12.0'
```

Comment:
The Maven quickstarts use `999-SNAPSHOT` as the version for it's quickstarts. I think we should do similar? @cescoffier  ?

Reply:
@stalep erm... sorry, but I don't think that's true? There is no `999-SNAPSHOT` and [the Maven quickstarts uses fixed version](https://github.com/quarkusio/quarkus-quickstarts/blob/master/getting-started/pom.xml#L10) (click link). If you agree, will you Resolve Conversation on this point?

Reply:
ah... my fault! I have a local commit where i have changed it to 999-SNAPSHOT, sorry ignore this